.env.test

```diff
@@ -4,5 +4,7 @@
 # For managing sensitive values and overrides use `/.env.test.local`
 #
 # Reference: https://github.com/bkeepers/dotenv#what-other-env-files-can-i-use
-
+DOMAIN=test.local
```

Comment:
Should a different `REDIS_URL` be specified here so the test suite doesn't overwrite keys in the development Redis instance?

Reply:
Good point. I think it would be good to use a Mock Redis.